internal/config/config.go

```diff
@@ -272,18 +272,20 @@ func (c *Config) SetJiraToken(token *oauth1.Token) {
 
 // configFile is a serializable representation of the current Viper configuration.
 type configFile struct {
-	LogLevel    string        `json:"log-level" mapstructure:"log-level"`
-	GithubToken string        `json:"github-token" mapstructure:"github-token"`
-	JiraUser    string        `json:"jira-user" mapstructure:"jira-user"`
-	JiraToken   string        `json:"jira-token" mapstructure:"jira-token"`
-	JiraSecret  string        `json:"jira-secret" mapstructure:"jira-secret"`
-	JiraKey     string        `json:"jira-private-key-path" mapstructure:"jira-private-key-path"`
-	JiraCKey    string        `json:"jira-consumer-key" mapstructure:"jira-consumer-key"`
-	RepoName    string        `json:"repo-name" mapstructure:"repo-name"`
-	JiraURI     string        `json:"jira-uri" mapstructure:"jira-uri"`
-	JiraProject string        `json:"jira-project" mapstructure:"jira-project"`
-	Since       string        `json:"since" mapstructure:"since"`
-	Timeout     time.Duration `json:"timeout" mapstructure:"timeout"`
+	LogLevel    string        `json:"log-level,omitempty" mapstructure:"log-level"`
+	DryRun      string        `json:"dry-run,omitempty" mapstructure:"dry-run"`
```

Comment:
See comment above about dry-run operations.
Also, the README suggests that this field is a boolean, but it is defined as a string here.
Prefer booleans in scenarios where string interpretation can lead to unexpected results e.g., `configFile.DryRun = "is-this-interpreted-as-true-or-false?"`

Reply:
The type is my mistake, should be bool of course